modules/ols-creating-lightspeed-custom-resource-file-using-cli.adoc

```diff
@@ -9,8 +9,6 @@ The Custom Resource (CR) file contains information that the Operator uses to dep
 
 .Prerequisites
 
-* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role. Alternatively, you are logged in to a user account that has permission to create a cluster-scoped custom resource file.
```

Comment:
Just curious if there's a reason why this was removed here, but not for https://80611--ocpdocs-pr.netlify.app/openshift-lightspeed/latest/configure/ols-configuring-openshift-lightspeed.html#ols-creating-lightspeed-custom-resource-file-using-web-console_ols-configuring-openshift-lightspeed ?

The procedure to create a CR I think would require the same permissions whether it's via CLI or not?

Reply:
@abrennan89 I removed it because when performing this task using the CLI you don't need access to the web console. Same for when performing this task using the console, you don't need the CLI. So, a suggestion during technical review was to remove the part that does not immediately apply.

Reply:
I understand that, but you might need to mention the cluster admin role part for the CLI procedures, just without the reference to the console, if those permissions are still required?